client/utils/index.ts

```diff
@@ -880,12 +880,14 @@ export const itemsEqual = (nextItem, currentItem) => {
 
     get(lhs, 'coverages', []).forEach(
         (coverage) => {
+            delete coverage.planning._scheduledTime;
```

Comment:
do you think deleting it won't impact other areas?

Reply:
@eos87 no it won't. The object we are deleting from is a deep clone of the actual object which is used to check if there's any new difference that needs to be saved.